### PR TITLE
Generic hotfixes

### PR DESCRIPTION
**Pull request by mp-17**
Round of hotfixes for bugs noticed while implementing new kernels.

### Fixed

- Fix masked `VSLIDEUP`. Use only the mask bits with index higher than the stride.
- Fix `SLDU` `issue_counter` modification upon new `VSLIDEUP` incoming instruction.
- Fix whole-register-move destination re-encoding.

### Changed

- Increase addrgen queue depth to four, to better hide the memory latency.

## Checklist

- [x] Automated tests pass
- [x] Changelog updated
- [x] Code style guideline is observed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - Fix stripmining condition in dispatcher
  - CVA6 tracks writes to floating-point scalar registers by the accelerator
  - Fixed de-synch bug in vector-FPU
+ - Fix masked VSLIDEUP. Use only the mask bits with index higher than the stride
+ - Fix SLDU issue_counter modification upon new VSLIDEUP incoming instruction
+ - Fix whole-register-move destination register re-encoding
+
```
Comment (suehtamacv) [inline CHANGELOG.md:41]: Unnecessary whitespace change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -44,6 +48,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - Vector integer reductions (`vredsum`, `vredmaxu`, `vredmax`, `vredminu`, `vredmin`, `vredand`, `vredor`, `vredxor`, `vwredsumu`, `vwredsum`)
  - Introduce the global hazard table in the main sequencer, to provide up-to-date information to the operand requesters about the status of the different dependant instructions
  - Integer and Floating-Point scalar move instructions (`vmv.x.s`, `vmv.s.x`, `vmv.f.s`, `vmv.s.f`)
+ - Increase addrgen queue depth to four, to better hide memory latency
```
Comment (suehtamacv) [inline CHANGELOG.md:51]: Changed, not added.

**Review by suehtamacv — APPROVED** [review]

**Comment by mp-17** [comment]
@suehtamacv thanks for the review, we can merge when the tests pass ;-)
